protocol-models/bin/generate-python-dataclasses.sh

```diff
@@ -22,13 +22,18 @@ mkdir -p "$ROOT_DIR/$OUTPUT_DIR"
 echo "# generated by generate-python-dataclasses" > "$ROOT_DIR/$OUTPUT_DIR"/__init__.py
 echo "name = 'models'" >> "$ROOT_DIR/$OUTPUT_DIR"/__init__.py
 
-for f in "$ROOT_DIR/$YAML_DIR"/*.yaml; do
+if [ -f $ROOT_DIR/$YAML_DIR/airbyte_protocol.yaml ]; then
+    echo ERROR: File airbyte_protocol.yaml exists in both $ROOT_DIR/$YAML_DIR and $ROOT_DIR/$YAML_DIR/v0. Only one is expected
+    exit 1
+fi
+
+for f in "$ROOT_DIR/$YAML_DIR"/*.yaml "$ROOT_DIR/$YAML_DIR"/v0/airbyte_protocol.yaml; do
   filename_wo_ext=$(basename "$f" | cut -d . -f 1)
   echo "from .$filename_wo_ext import *" >> "$ROOT_DIR/$OUTPUT_DIR"/__init__.py
 
   datamodel-codegen \
     --input-file-type jsonschema \
-    --input "$ROOT_DIR/$YAML_DIR/$filename_wo_ext.yaml" \
+    --input "$f" \
```

Comment:
this would no longer generate the v0 file right? do we need that for anything else? (not sure if there are other consumers of the protocol)

Reply:
oh maybe the output is still the same?

Reply:
My understanding is that it never generated the V0 files. However when I compared v0 and v2 airbyte_protocol.yaml files, they were the same (except some formatting and comments differing)